docs/UserGuide.md

```diff
@@ -245,32 +261,54 @@ For a comprehensive deep-dive into the undo command, please refer to the [Implem
 
 </box>
 
-Examples:
-_IMAGE_BEFORE and IMAGE_AFTER TBA_
-
 ### Redoing a command : `redo`
 
 _This command allows you to redo your command, reversing any changes that were previously undone._
 
 Format: `redo`
 
+**Walkthrough:**
+
+The screenshots below are what you would expect when using the `redo` command.
+
+In this example, after undoing an update of my client's name from `Alex Yeoh` to `alex yee`, we can see that the current client list has his name as `Alex Yeoh`
+&nbsp;
+
+<img src="images/user-guide/redo_command_before.png" alt="Redo command example" />
+&nbsp;
+
+As seen in the second image, after using the `redo` command, the change is redone, and the client list shows the client's name as `alex yee`
+&nbsp;
+
+<img src="images/user-guide/redo_command_after.png" alt="Redo command example" />
```

Comment:
A caption should be added below the image to describe what is happening (see delete for example).